From afdd5ae3eeb44381f906b6227422373d4af2811d Mon Sep 17 00:00:00 2001
From: Vishnu Banavath <vishnu.banavath@arm.com>
Date: Wed, 21 Sep 2022 15:54:14 +0100
Subject: [PATCH] arm64: defconfig: disable config options that does not apply 
 anymore

Following config options should be not set to be more accurate and
works with build system like yocto
CONFIG_BT_HCIBTUSB
CONFIG_BT_HCIBTUSB_MTK
CONFIG_BT_HCIUART_MRVL
CONFIG_BT_MRVL
CONFIG_BT_MRVL_SDIO
CONFIG_BT_QCOMSMD
CONFIG_BT_NXPUART

Upstream-Status: Pending [not submitted upstream yet]
Signed-off-by: Adam Johnston <adam.johnston@arm.com>
Signed-off-by: Vishnu Banavath <vishnu.banavath@arm.com>
Signed-off-by: Xueliang Zhong <xueliang.zhong@arm.com>
Signed-off-by: Anusmita Dutta Mazumder <anusmita.duttamazumder@arm.com>
---
 arch/arm64/configs/defconfig | 14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/arch/arm64/configs/defconfig b/arch/arm64/configs/defconfig
index d2b5208eb55d..b3028113de25 100644
--- a/arch/arm64/configs/defconfig
+++ b/arch/arm64/configs/defconfig
@@ -186,17 +186,17 @@ CONFIG_BT_HIDP=m
 # CONFIG_BT_LE is not set
 CONFIG_BT_LEDS=y
 # CONFIG_BT_DEBUGFS is not set
-CONFIG_BT_HCIBTUSB=m
-CONFIG_BT_HCIBTUSB_MTK=y
+# CONFIG_BT_HCIBTUSB is not set
+# CONFIG_BT_HCIBTUSB_MTK is not set
 CONFIG_BT_HCIUART=m
 CONFIG_BT_HCIUART_LL=y
 CONFIG_BT_HCIUART_BCM=y
 CONFIG_BT_HCIUART_QCA=y
-CONFIG_BT_HCIUART_MRVL=y
-CONFIG_BT_MRVL=m
-CONFIG_BT_MRVL_SDIO=m
-CONFIG_BT_QCOMSMD=m
-CONFIG_BT_NXPUART=m
+# CONFIG_BT_HCIUART_MRVL is not set
+# CONFIG_BT_MRVL is not set
+# CONFIG_BT_MRVL_SDIO is not set
+# CONFIG_BT_QCOMSMD is not set
+# CONFIG_BT_NXPUART is not set
 CONFIG_CFG80211=m
 CONFIG_MAC80211=m
 CONFIG_MAC80211_LEDS=y
-- 
2.38.1

